main.cpp

```diff
@@ -8,38 +8,37 @@ int main() {
     Student student2{"Anna", "Kowalska", "Katowice", "wesoła", "1", 4321, "98876543210", Gender::Female};
     Student student3{"Justyna", "Dunka", "Poznan", "wesoła", "1", 4321, "98876543210", Gender::Female};
 
-     Database db;
+    Database db;
     db.printAll();
     db.addStudent(student);
     db.addStudent(student2);
     db.addStudent(student3);
 
     db.printAll();
 
-    std::cout<<"---------SORT BY PESEL-----------\n";
+    std::cout << "---------SORT BY PESEL-----------\n";
     db.sortByPesel();
     db.printAll();
 
-    std::cout<<"---------SORT BY SURENAME-----------\n";
-    db.sortbySurename();
+    std::cout << "---------SORT BY SURENAME-----------\n";
+    db.sortBySurName();
     db.printAll();
 
-    std::cout<<"---------SAVE EXSISTS RECORD TO FILE-----------\n";
+    std::cout << "---------SAVE EXSISTS RECORD TO FILE-----------\n";
     db.saveToFile("db.txt");
-    
+
     db.deleteByPesel("12345678901");
 
     db.printAll();
     auto result = db.searchByPesel("98876543210");
     if (!result.empty()) {
-           for (const auto& el : result) {
-                std::cout << el.getFirstName() << "  ";
-                std::cout << el.getSurName() << ", ";
-                std::cout << el.getAddress() << " | Pesel: ";
-                std::cout << el.getPesel() << '\n';
-           } 
-    }
-    else {
+        for (const auto& el : result) {
+            std::cout << el.getFirstName() << "  ";
+            std::cout << el.getSurName() << ", ";
+            std::cout << el.getAddress() << " | Pesel: ";
+            std::cout << el.getPesel() << '\n';
+        }
+    } else {
         std::cout << "Not found Student with this pesel\n";
     }
```

Comment:
maybe extract to a new function that takes a result argument and returns a text string to avoid cluttering main function

Reply:
Maybe just use overriden operator <<: 
std::cout << el << '\n';
Will be consider on next PR.